rakefile

```diff
@@ -15,13 +15,50 @@ rototiller_task :performance do
   Rake::Task["performance_provision_with_abs"].execute unless ENV['ABS_RESOURCE_HOSTS'] ||
       (ENV['BEAKER_HOSTS'] &&
           ENV['BEAKER_HOSTS'] != 'config/beaker_hosts/pe-perf-test.cfg' &&
-          ENV['BEAKER_HOSTS'] != 'config/beaker_hosts/foss-perf-test.cfg')
+          ENV['BEAKER_HOSTS'] != 'config/beaker_hosts/foss-perf-test.cfg') &&
+          ENV['BEAKER_HOSTS'] != 'config/beaker_hosts/opsworks-perf-test.cfg'
   Rake::Task["performance_without_provision"].execute
   Rake::Task["performance_deprovision_with_abs"].execute unless ENV['BEAKER_PRESERVE_HOSTS'] == 'always' ||
       ((@beaker_cmd.nil? || @beaker_cmd.result.exit_code != 0)&&
           ENV['BEAKER_PRESERVE_HOSTS'] == 'onfail')
 end
 
+desc 'Provision systems using ABS (or use already set ABS_RESOURCE_HOSTS) and then execute beaker performance setup ' +
+    'and tests for opsworks'
+rototiller_task :opsworks_performance do
+  raise "The test must be Opsworks.rb, or '' if you plan to execute tests later" unless
+    ["", "tests/OpsWorks.rb"].include? ENV["BEAKER_TESTS"]
```

Comment:
Why not do an if not set, set to OpsWorks.  From the name of the task, it's pretty obvious what they want to do...

Reply:
I was thinking that they may not want to run any tests and just set stuff up, but you're right, for that they would set it to an empty string, if it's not set at all, we should set it.